### PR TITLE
[1LP][RFR] Adding manual test of services RFE

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================

{{pytest: --manual --long-running --use-template-cache --collect-only cfme/tests/services/test_catalog_item.py cfme/tests/services/test_generic_service_catalogs.py cfme/tests/services/test_myservice.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit dbfbf978dffbdb1f56a54e56432ce17030f07c99:

`cfme/tests/services/test_catalog_item.py`:
- :heavy_exclamation_mark: Line 409:43: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 427:30: **W291** *trailing whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_myservice.py [hunk 1]
```diff
@@ -185,3 +185,23 @@ def test_retire_on_date_for_multiple_service():
...
+@pytest.mark.ignore_stream("5.10")
+def test_service_state():
+    """
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/4h
+        startsin: 5.11
+        testSteps:
+            1. Create catalog,catalog item
```
Comment (digitronik) [inline cfme/tests/services/test_myservice.py]: Required:
need a `space`

**Comment by digitronik** [comment]
@niyazRedhat I understand its `5.11` tests and you added these test as per BZ. I won't like to force you for more specific steps still if possible you can add more info or `Dev PR` links to it.

**Comment by niyazRedhat** [comment]
@digitronik These RFE BZ is still in assign state. I will add  more details and PR link once developer will raise PR

**Pull request merged**